xsens-tf-bridge/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 2.8.9)
+find_package(YARP REQUIRED)
+find_package(iDynTree REQUIRED)
+include_directories(include)
+
+yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/thrift/XsensFrame.thrift" "${CMAKE_CURRENT_BINARY_DIR}/autogenerated/data" THRIFT_SOURCES THRIFT_HEADERS THRIFT_INCLUDE_DIRS)
+yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/thrift/XsensDriverService.thrift" "${CMAKE_CURRENT_BINARY_DIR}/autogenerated/services" THRIFT_SERVICE_SOURCES THRIFT_SERVICE_HEADERS THRIFT_SERVICE_INCLUDE_DIRS)
+
+add_library(thriftLibrary ${THRIFT_SOURCES} ${THRIFT_HEADERS} ${THRIFT_SERVICE_SOURCES} ${THRIFT_SERVICE_HEADERS})
+target_link_libraries(thriftLibrary ${YARP_LIBRARIES})
+target_include_directories(thriftLibrary PUBLIC ${THRIFT_INCLUDE_DIRS} ${THRIFT_SERVICE_INCLUDE_DIRS})
```

Comment:
Thrifts generetes a lot of warnings.
You can add `SYSTEM` to the directive:
`target_include_directories(thriftLibrary SYSTEM PUBLIC ...`